src/main/java/org/traccar/protocol/TotemProtocolEncoder.java

```diff
@@ -1,12 +1,12 @@
 /*
  * Copyright 2015 Irving Gonzalez
- * Copyright 2015 - 2019 Anton Tananaev (anton@traccar.org)
+ * Copyright 2016 - 2019 Anton Tananaev (anton@traccar.org)
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
- *      http://www.apache.org/licenses/LICENSE-2.0
+ *     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Why are there changes to the header?

Reply:
I based this off of another Encoder at the start - and forgot about the changes to the header/how it will look. I will fix it back like it was in this file at the start :)